merge_duplicate_contacts/wizard/merge_contact.py

```diff
@@ -0,0 +1,935 @@
+import itertools
+import logging
+import operator
+from ast import literal_eval
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError, ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    def open_wizard_action(self):
+        context = {}
+        data = {
```

Comment:
If these are fields defaults, remove entries values that are ... default.

Just add a _prepare_...() function if one want to override it.